client/src/store/reducers/ImagesReducer.ts

```diff
@@ -0,0 +1,32 @@
+import { AnyAction } from 'redux';
+import { LOAD_IMAGES } from '../actions';
+
+export interface Image {
+  id: number;
+  name: string;
```

Comment:
name is not needed right now, let's remove stuff we don't need

Reply:
can you replace the name with *fileName* ? I will add that

Reply:
Will change in upcoming PR so this is not blocked.